pkgs/applications/video/gnome-mpv/default.nix

```diff
@@ -5,21 +5,24 @@
 
 stdenv.mkDerivation rec {
   name = "gnome-mpv-${version}";
```

Comment:
```suggestion
  pname = "gnome-mpv";
```

Reply:
Fixed. I can deduce how `pname` works from looking at other derivations, but I couldn't find any documentation on it.

Reply:
Opened a PR for `pname` documentation https://github.com/NixOS/nixpkgs/pull/63196